src/main/resources/application.yaml

```diff
@@ -34,6 +34,13 @@ server:
 file:
   path: "files"
 
+ses:
```

Comment:
Add additional prefix `aws` before `ses`. Later it will be useful `aws` will contain all services (ses, s3).